### ISSUE TITLE
Critical error on plugin desactivate

### ISSUE THREAD

**Issue by Maxence1502** [crowdsecurity/cs-wordpress-bouncer#40]
![image](https://user-images.githubusercontent.com/42026318/127879764-dffe5b90-a7cb-4f42-8bab-56af097dac44.png)

=============
```
An error of type E_ERROR was caused in line 311 of the file /var/www/developpement/wp-content/plugins/crowdsec/vendor/crowdsec/bouncer/src/ApiCache.php. Error message: Uncaught Error: Call to a member function toString() on null in /var/www/developpement/wp-content/plugins/crowdsec/vendor/crowdsec/bouncer/src/ApiCache.php:311
Stack trace:
#0 /var/www/developpement/wp-content/plugins/crowdsec/vendor/crowdsec/bouncer/src/ApiCache.php(446): CrowdSecBouncer\ApiCache->saveRemediations()
#1 /var/www/developpement/wp-content/plugins/crowdsec/vendor/crowdsec/bouncer/src/Bouncer.php(170): CrowdSecBouncer\ApiCache->warmUp()
#2 /var/www/developpement/wp-content/plugins/crowdsec/inc/admin/init.php(40): CrowdSecBouncer\Bouncer->warmBlocklistCacheUp()
#3 /var/www/developpement/wp-content/plugins/crowdsec/inc/plugin-setup.php(55): clearBouncerCacheInAdminPage()
#4 /var/www/developpement/wp-includes/class-wp-hook.php(303): deactivate_crowdsec_plugin()
#5 /var/www/developpement/wp-includes/class-wp-hook.php(327): WP_Hook->apply_filters()
#6 /var/www/developpement/wp-includes/plugin.php(470): WP_Hook->do_action()
#7 /var/www/developpement/wp-admin/includes/plugin.php(821): do_action()
#8 /va

```

**Comment by mobula9** [comment]
Thx @Maxence1502 , I'll investigate.

**Comment by Maxence1502** [comment]
Hey! Any update ? :p

**Comment by Maxence1502** [comment]
Up please

**Comment by mobula9** [comment]
Hey @Maxence1502, I'm back from holidays, many things to do but I will try to check that this week !

**Comment by Maxence1502** [comment]
Thanks !

**Comment by mobula9** [comment]
After investigating the code,  it seems that in the decisions list of your LAPI, there is one or many decisions with a scope "Ip" but the "Value" of the decision is not a valid IP.

**Comment by mobula9** [comment]
I think this use case is an edge case.

Therefore,  for this kind of case I can add a check o let the process continue ignoring not valid ip values in decisions scoped as "Ip".

**Comment by Maxence1502** [comment]
How can I check this? There are thousands of decisions in LAPI.
They have all been added automatically, maybe there is a problem in crowdsec too at this point

**Comment by Maxence1502** [comment]
Yes you can update the plugin by adding a check to avoid errors, it will be already solved :p

**Comment by mobula9** [comment]
If they were added automatically, indeed there must be a problem upstream as well. We will also check this.

**Comment by Maxence1502** [comment]
Ok no problem, keep me informed about the updates

**Comment by mobula9** [comment]
I just contacted you in private on Gitter to investigate together the problem on your own logs ;-)

**Comment by Maxence1502** [comment]
Ok :)

**Comment by mobula9** [comment]
Fixed with the v1.0.6
